scripts/workspace-conversion/workspace-to-dataset.py

```diff
@@ -0,0 +1,296 @@
+# ------------------------------------------------------------------------------
+# Workspace to Dataset
+#
+# This script converts terra workspaces attributes into a potential catalog entry.
+#
+# Before Running:
+#   python3 -m venv .env
+#   source .env/bin/activate
+#   pip install --upgrade pip
+#   python3 -m pip install requests
+#
+#   Authorization: Access the password in the vault using the appropriate command:
+#      (you will need this for the auth dialog that the script will pull up)
+#   vault login -method=github token=$(cat ~/.github-token)
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/userAdmin
+#
+# Run:
+#   `python3 workspace-to-dataset.py`
+#   `GCLOUD_USER={your-email} python3 generate-tdr-snapshot-metadata.py`
+#
+# Environment Variables:
+#   GCLOUD_USER: user account email
+#   CATALOG_SERVICE_URL: url for the catalog service to update the updata in
+#   DATA_REPO_URL: url for the data repo to use to extract testing snapshot information
+# ------------------------------------------------------------------------------
+import json
+import os, subprocess, sys
+import requests
+
+urlRoot = os.environ.get("RAWLS_URL") or "https://rawls.dsde-prod.broadinstitute.org"
+urlWorkspace = f"{urlRoot}/api/workspaces"
+workspaceNamespace = os.environ.get("WORKSPACE_NAMESPACE")
+workspaceName = os.environ.get("WORKSPACE_NAME")
+user = os.environ.get("GCLOUD_USER") or "datacatalogadmin@test.firecloud.org"
+
+
+def logResponse(response, message):
+    if 200 <= response.status_code and response.status_code < 300:
```

Comment:
I think `ok` is more idiomatic way to write this
```suggestion
    if response.ok:
```